liquidprompt

```diff
@@ -1133,6 +1135,28 @@ _lp_python_env_color() {
     lp_python_env_color="[${LP_COLOR_VIRTUALENV}${lp_python_env}${NO_COL}]"
 }
 
+_lp_ruby_env() {
+    (( LP_ENABLE_RUBY_VENV )) || return 2
+
+    local ret
+
+    if command -v rvm-prompt >/dev/null ; then
+        __lp_escape "$(eval rvm-prompt $LP_RUBY_RVM_PROMPT_OPTIONS)"
+        lp_ruby_env=$ret
+    elif command -v rbenv >/dev/null ; then
+        __lp_escape "$(rbenv version | sed -e 's/ (set.*$//' -e 's/^ruby-//')"
```

Comment:
Running `sed` is slow. Since this looks like a simple pattern removal, could this be done with parameter expansion instead?
```suggestion
        lp_ruby_env=$(rbenv version)
        lp_ruby_env=${lp_ruby_env%% (set*}
        __lp_escape "${lp_ruby_env#ruby-}"
```

Reply:
I must admit I never use rbenv myself. This line comes directly from the first comment of the related issue. I’ll try to review how rbenv works and see if I can improve it.

Reply:
I take your proposal and commited it in b5fe8d6d5b558fd163155fd68dc721213bfbd74e

I kept only the removal of the ` (set…)` part of the output, as the `ruby-` indication may be usefull (to discriminate against jruby, truffle ruby…) and more homogeneous with default RVM output.